docs/js.md

```diff
@@ -0,0 +1,162 @@
+### [Primative: Null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null)
+
+* _symbol_: `null`
+* _pronunciation_: "null", "null value", "is null"
+* _examples_:
+```javascript
+null // how exciting
+```
+
+### [Primative: Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)
+
+*_symbol_: 'true' 'false'
+*_pronunciation_"boo-lee-an"
+*_examples_:
+```javascript
+50 < 100 == true
+50 > 100 == false
+
+('meh' != meh) == true
+```
+
+### [Primative: Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)
+
+*_symbol_: 'The numeric value of the object being created.'
```

Comment:
Might there be additional symbols that we can use to represent numbers? You might at least list the "numeric values" for reference. You should have more to add here after today's lecture. To exponents and beyond!